ci-operator/step-registry/hypershift/setup-nested-management-cluster/hypershift-setup-nested-management-cluster-chain.yaml

```diff
@@ -44,23 +49,31 @@ chain:
       CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)-mgmt"
       
       echo "Creating management cluster ${CLUSTER_NAME} with $((${HYPERSHIFT_NODE_COUNT} * 3)) nodes"
-      bin/hypershift create cluster aws \
-        --pull-secret=/etc/ci-pull-credentials/.dockerconfigjson \
-        --aws-creds=${AWS_GUEST_INFRA_CREDENTIALS_FILE} \
-        --additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)" \
-        --name=${CLUSTER_NAME} \
-        --region=us-east-1 \
-        --infra-id=${CLUSTER_NAME} \
-        --base-domain=${DOMAIN} \
-        --instance-type=${HYPERSHIFT_INSTANCE_TYPE} \
-        --zones=${HYPERSHIFT_ZONES} \
-        --service-cidr=172.29.0.0/16 \
-        --cluster-cidr=10.124.0.0/14 \
-        --node-pool-replicas=${HYPERSHIFT_NODE_COUNT} \
-        --node-selector=hypershift.openshift.io/control-plane=true \
-        --release-image=${HYPERSHIFT_HC_RELEASE_IMAGE} \
-        --release-stream=${DEFAULT_RELEASE_STREAM} \
-        --namespace=${HYPERSHIFT_NAMESPACE}
+      COMMAND=(
+        bin/hypershift create cluster aws \
+          --pull-secret=/etc/ci-pull-credentials/.dockerconfigjson \
+          --aws-creds=${AWS_GUEST_INFRA_CREDENTIALS_FILE} \
+          --additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)" \
+          --name=${CLUSTER_NAME} \
+          --region=us-east-1 \
+          --infra-id=${CLUSTER_NAME} \
+          --base-domain=${DOMAIN} \
+          --instance-type=${HYPERSHIFT_INSTANCE_TYPE} \
+          --zones=${HYPERSHIFT_ZONES} \
+          --service-cidr=172.29.0.0/16 \
+          --cluster-cidr=10.124.0.0/14 \
+          --node-pool-replicas=${HYPERSHIFT_NODE_COUNT} \
+          --node-selector=hypershift.openshift.io/control-plane=true \
+          --release-image=${HYPERSHIFT_HC_RELEASE_IMAGE} \
+          --release-stream=${DEFAULT_RELEASE_STREAM} \
+          --namespace=${HYPERSHIFT_NAMESPACE}
+      )
+
+      if [[ $FEATURE_SET == "TechPreviewNoUpgrade" ]]; then
```

Comment:
The issue is here.  We don't want `TechPreviewNoUpgrade` on the prow job mgmt cluster, only on the test cluster created in the `hypershift-aws-create` chain.

Reply:
ok, I am trying it now. IIRC this was my first effort (the hacky way) and I found that the install would work, but tech preview features were still not enabled.

let's see how the rehearsal looks now.

Reply:
I remembered incorrectly. this does work. [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/56065/rehearse-56065-pull-ci-openshift-ovn-kubernetes-release-4.18-e2e-aws-ovn-hypershift-conformance-techpreview/1849544643998388224/artifacts/e2e-aws-ovn-hypershift-conformance-techpreview/dump/artifacts/hostedcluster-903a01a56eae65bd1183/cluster-scoped-resources/config.openshift.io/featuregates.yaml) is the featuregates for the hosted cluster.